js/slider/Slider.stories.svelte

```diff
@@ -0,0 +1,48 @@
+<script>
+	import { Meta, Template, Story } from "@storybook/addon-svelte-csf";
+	import Slider from "./interactive";
+</script>
+
+<Meta title="Components/Slider" component={Slider} />
+
+<Template let:args>
+	<Slider {...args} />
+</Template>
+
+<Story
+	name="Slider with min 0 and max 100"
+	args={{
+		minimum: 0,
+		maximum: 100
+	}}
+/>
+
+<Story
+	name="Slider with step of 10"
+	args={{
+		step: 10
+	}}
+/>
+
+<Story
+	name="Slider in disabled state"
+	args={{
+		mode: "static"
```

Comment:
Nit: this one isn't static actually. I think its because you're importing Slider from "./interactive";